lib/entice/logic/MapInstance.ex

```diff
@@ -0,0 +1,67 @@
+defmodule Entice.Logic.MapInstance do
+  alias Entice.Entity
+  alias Entice.Logic.MapInstance
+
+  defstruct(
+    players: [],
```

Comment:
i'd say this can be a simple integer counter ^^
cause we currently probably dont need to be able to reference all players individually